src/components/templates/PartyMap/PartyMapPage.tsx

```diff
@@ -93,21 +97,33 @@ const PartyMap = () => {
           <CountDown startUtcSeconds={venue.start_utc_seconds} />
         </div>
         <div className="row">
-          <Map config={venue} attendances={attendances} />
+          <Map
+            config={venue}
+            attendances={attendances}
+            setSelectedRoom={setSelectedRoom}
+          />
         </div>
         <div className="row">
           <div className="col">
             <RoomList
               startUtcSeconds={venue.start_utc_seconds}
               rooms={venue.rooms}
               attendances={attendances}
+              setSelectedRoom={setSelectedRoom}
             />
           </div>
           <div className="col-5 chat-wrapper">
             <Chatbox />
           </div>
         </div>
       </div>
+      {selectedRoom && (
+        <RoomModal
+          show={true}
```

Comment:
remove ={true}

Reply:
by using `selectedRoom &&` instead of `show={!!selectedRoom}`, when selectedRoom is changed from true to false do we still see the Modal fade away with a nice animation or does it just disappear?

Reply:
it just disappears nice catch